Makefile

```diff
@@ -482,6 +482,10 @@ lint: tools go-lint doc-lint misspell yamllint ## Run linting and misspell check
 misspell:
 	hack/check/misspell.sh
 
+actionlint:
+	go install github.com/rhysd/actionlint/cmd/actionlint@latest
```

Comment:
`latest` (v1.6.17) seems to be croaking with go 1.17. I did verify version v1.6.16 works. Should we switch to it?

Also nice catch and and thanks from bringing the invalid property issue to attention, @seemiller !

I have a PR out to fix it: https://github.com/vmware-tanzu/tanzu-framework/pull/3360
Once that is addressed, we can remove L487

Reply:
#3360 is merged